django/db/models/manager.py

```diff
@@ -81,11 +82,10 @@ def check(self, **kwargs):
     @classmethod
     def _get_queryset_methods(cls, queryset_class):
         def create_method(name, method):
+            @wraps(method)
             def manager_method(self, *args, **kwargs):
                 return getattr(self.get_queryset(), name)(*args, **kwargs)
 
-            manager_method.__name__ = method.__name__
-            manager_method.__doc__ = method.__doc__
```

Comment:
Is it currently being tested? If not, it will be good to add tests in a separate commit to confirm that we are not introducing any regression.

Reply:
I'd add them to the `tests.basic.tests.ManagerTest`:

https://github.com/django/django/blob/90aabd730a2a434c227faf8a927b0e2ccd67e291/tests/basic/tests.py#L659